ui/page/root/receive_page.go

```diff
@@ -168,170 +196,152 @@ func (pg *ReceivePage) generateQRForAddress() {
 	pg.qrImage = &imgdec
 }
 
+func (pg *ReceivePage) getSelectedWalletLogo() *cryptomaterial.Image {
+	pg.selectedWallet.GetAssetType()
+	switch pg.selectedWallet.GetAssetType() {
+	case utils.BTCWalletAsset:
+		return pg.Theme.Icons.CircleBTC
+	case utils.DCRWalletAsset:
+		return pg.Theme.Icons.CircleDCR
+	case utils.LTCWalletAsset:
+		return pg.Theme.Icons.CircleLTC
+	default:
+		return nil
+	}
+}
+
 // Layout draws the page UI components into the provided C
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *ReceivePage) Layout(gtx C) D {
-	pg.handleCopyEvent(gtx)
-	pg.pageBackdropLayout(gtx)
-
-	if pg.Load.IsMobileView() {
-		return pg.layoutMobile(gtx)
+	if pg.modalLayout == nil {
+		return pg.contentLayout(gtx)
 	}
-	return pg.layoutDesktop(gtx)
+	var modalWidth float32 = 450
```

Comment:
Is this supposed to be a `unit.DP`? Can use the `values.MarginPadding450` var.

Reply:
It seems that using float32 constants is not really okay but we should fix it later, it is not within the scope of this PR